src/garage/envs/bullet/bullet_env.py

```diff
@@ -128,18 +222,19 @@ def __getstate__(self):
             dict: The instance’s __init__() arguments
 
         """
-        env = self._env.env
+        env = self.env.env
 
         # Extract constructor signature
         sig = inspect.signature(env.__init__)
         args = {}
         param_names = list(sig.parameters.keys())
 
         # Hard fix for args/private variable name inconsistency
-        if isinstance(env, (MinitaurBulletEnv, MinitaurBulletDuckEnv)):
+        if env.spec.id.find('MinitaurBulletEnv') >= 0 or env.spec.id.find(
```

Comment:
why not `if 'MinitaurBulletEnv' in env.spec.id or MinitaurBulletDuckEnv' in env.spec.id:`?